packages/harmony/src/components/text/Text.tsx

```diff
@@ -85,8 +85,8 @@ export const Text = forwardRef(
         overflow: 'hidden',
         display: '-webkit-box',
         lineClamp: `${maxLines}`,
-        '-webkit-line-clamp': `${maxLines}`,
-        '-webkit-box-orient': 'vertical'
+        WebkitLineClamp: `${maxLines}`,
```

Comment:
i thought that i tested this and it didn't work? does this work now?

Reply:
Yup it seems to work. I changed it because there was an error being thrown in the console about kebab case properties